### ISSUE TITLE
Can't cancel Conditional Orders on Bybit

### ISSUE THREAD

**Issue by maciekzygmunt** [ccxt/ccxt#13075]
I can't cancel conditional orders on Bybit. Tried to use _cancel_all_orders_ but it cancels only limit orders.

- Programming Language version: Python 3.9

```
client.cancel_all_orders('BTCUSDT')
```

**Comment by Dan-krm** [comment]
I've submitted a PR that will enable using cancelAllOrders for conditional orders on Bybit

**Comment by kroitor** [comment]
@maciekzygmunt it is available in CCXT version 1.81.50+

How to use:

```Python
client.cancel_all_orders('BTCUSDT', {'stop': true})
```
